### DISCUSSION TITLE
TCP server connections limit

### DISCUSSION THREAD

**Discussion by rudolfninja** [jchristn/SuperSimpleTcp#175] [Q&A]
Did you run kind of stress test of the server to know what is the maximum number of simultaneously connected clients to one server instance? Having a machine with Windows Server 2016 and 128GB of RAM is it possible to handle 100k of simultaneously connected clients?

**Comment by jchristn** [comment]
Hi @rudolfninja I do not.  There are too many variables to consider and I lack the hardware/software necessary to do it (also I haven't really had the need to do this).  Hopefully someone else has had an opportunity to do this kind of test and can share their results.
